### ISSUE TITLE
Eframe unusable when rendering with WGPU backend on Wayland

### ISSUE THREAD

**Issue by anantnrg** [emilk/egui#3924]
**Describe the bug**

When running `eframe` with the `wgpu` and `wayland` features enabled, the result window is unresponsive to any mouse or keyboard events. Running with `glow` works perfectly fine as intended.

**To Reproduce**
1. Clone `emilk/eframe_template` repo.
2. Open `Cargo.toml`
3. Change `glow` in `eframe` `features` to `wgpu`
4. Add `wayland` feature to `eframe`
5. Run with `cargo run`
6. Try clicking or typing anything.

**Expected behavior**
App should work as expected like when using the Glow backend.

**Desktop (please complete the following information):**
 - OS: EndeavourOS 6.7.2-arch-1

**Comment by Friz64** [comment]
Are you on NVIDIA, by chance? I presume this is https://github.com/gfx-rs/wgpu/issues/4775. Will most likely be fixed in the next major wgpu release (0.20), or when NVIDIA decides to fix their drivers.

**Comment by anantnrg** [comment]
@Friz64 yes, I have an Nvidia GTX 1060. I'll test it once the new release of `wgpu` comes out. Maybe that'll solve it.

**Comment by Wumpf** [comment]
@anantnrg it's been a few updates by now, did you try again?

**Comment by anantnrg** [comment]
@Wumpf sorry im late, but no i did not try again.

**Comment by Wumpf** [comment]
@anantnrg mind giving it another spin with the latest egui version? :)
Also if this still doesn't work, can you please provide any log output generated by `RUST_LOG=trace`

**Comment by anantnrg** [comment]
@Wumpf sadly, im on windows right now (ue5 works better), and dont have a working linux install. So i wont be able to test it again. sorry :(

**Comment by Wumpf** [comment]
Closing this then due to lack of information & repro
